src/main/java/mate/academy/internetshop/controller/order/OrderController.java

```diff
@@ -14,7 +14,7 @@
 import mate.academy.internetshop.service.UserService;
 
 public class OrderController extends HttpServlet {
```

Comment:
```suggestion
public class CompleteOrderController extends HttpServlet {
```
давай трішки підправимо неймінг, щоб було зрозуміліше, за що відповідає цей контроллер

Reply:
Done